utils/format.sh

```diff
@@ -36,6 +36,10 @@ fix_format() {
     -o -name \*.cpp -print0 \
     -o -name \*.cl -print0 \
   | xargs -0 -P8 -n1 $CLANG_COMMAND -i;
+
+  find torch_glow/tests utils \
```

Comment:
Why not everything in torch_glow? There's also examples, setup.py, utils, etc

Reply:
That also formats things like `torch_glow//.eggs/py-1.8.0-py3.7.egg/py/_version.py`

Reply:
:(
those should are basically just build artifacts though

Reply:
autopep8 has an `exclude` configuration, maybe that can be used?

Reply:
It's possible that I don't know how to write globs but I can't get it to work :(

Even `autopep8 -i --exclude *.py test.py` still results in `test.py` being formatted.

Reply:
IMO formatting the eggs is fine.
Looks like `--exclude` thing doesn't work on explicitly listed files, but does work if you use `-r`.  Maybe instead of using `find|xargs` we should just do `autopep8 -r -j 8 -i -a .`

Reply:
I modified the command in the formatting script to
```
autopep8 -i -r -a -a -j -1 --exclude="*.eggs" torch_glow utils
```